website/__tests__/redux/integration/ElementIntegrationTests.jsx

```diff
@@ -61,26 +70,24 @@ describe('PositionsIntegrationTests.', () => {
     const updatedLoc = { x: 33, y: 88 };
 
     const unsubscribe = testStore.subscribe(() => {
```

Comment:
Not sure how to make both this one and the one below work with done() since it seems we are done once we call the function once. I expect that the function below will execute after this one so I just put done() in that one.